src/modules/users/entities/user.entity.ts

```diff
@@ -1,17 +1,24 @@
 import { DataTypes } from 'sequelize';
-import { Table, Column, Model, PrimaryKey, BelongsToMany } from 'sequelize-typescript';
+import {
+	Table,
+	Column,
+	Model,
+	PrimaryKey,
+	BelongsToMany,
+	BeforeCreate,
+} from 'sequelize-typescript';
 import { Role } from 'src/modules/roles/entities/role.entity';
-import { UserRole } from 'src/modules/roles/entities/userRole.entitty';
+import { UserRole } from 'src/modules/roles/entities/userRole.entity';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
import styles say that the external imorts on the top and internal on the bottom.